lib/charms/ovn_charm.py

```diff
@@ -473,6 +474,25 @@ def resume(self):
             os.path.join(ch_core.hookenv.charm_dir(), 'hooks/config-changed'),
         )
 
+    def custom_assess_status_last_check(self):
+        """Check if has valid bridge config and set to blocked if is invalid.
+
+        Returns (None, None) if the interfaces are okay, or a status, message
+        if the config is invalid.
+
+        :returns status & message info
+        :rtype: (status, message) or (None, None)
+        """
+
+        if not self.valid_config:
+            message = ('Wrong format. '
```

Comment:
Should this message mention which configuration option is subject to the bad format?